File: Modul 6/Soal 1/PRAK601-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>

void matriks(int baris, int kolom) {
    int matriks[baris][kolom];

    for(int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }

    printf("\n");
    for (int i = 0; i < baris; i++) {
        for(int j = 0;j < kolom; j++) {
            printf("%d  ", matriks[i][j]);
        }
        printf("\n");
    }
}

int main() {

    int baris, kolom;
    printf("Masukkan Jumlah Baris: ");
    scanf("%d", &baris);
    printf("Masukkan Jumlah Kolom: ");
    scanf("%d", &kolom);
    matriks(baris, kolom);
    return 0;
}

File: Modul 6/Soal 1/PRAK601-2410817220018-ANGGRAENI DWI ZAHRA.py
def matriks(baris, kolom):
    matriks = []            #masukkan input satu per satu
    for i in range(baris):
        num = []
        for j in range(kolom):
            num.append(int(input()))
        matriks.append(num)
    print("")
    for i in range(baris):
        for j in range(kolom):
            print(matriks[i][j], end = " ")
        print("")

baris = int(input("Masukkan Jumlah Baris: "))
kolom = int(input("Masukkan Jumlah Kolom: "))
matriks(baris, kolom)